src/containers/ToDoList/ToDoList.jsx

```diff
@@ -83,9 +92,16 @@ export const ToDoList = () => {
       id: uuidv4(),
       text: newTaskInput,
       checked: false,
+      dateAdded: new Date().toLocaleDateString("pl-PL", {
+        day: "2-digit",
+        month: "2-digit",
+        year: "numeric",
+      }),
```

Comment:
Most of the time, whenever we store dates, the date is stored in the most generic format (meaning not specific to the country/region). What I'd advise you to do here, is to store the date in ISO format (so this exactly what just `new Date()` returns). You format the date always when you want to show it. But when it comes to storing, you keep the most confusing computer-friendly piece of information ;-)